src/main.rs

```diff
@@ -291,12 +288,12 @@ struct CheckRelease {
     verbosity: clap_verbosity_flag::Verbosity<clap_verbosity_flag::InfoLevel>,
 
     /// Whether to print colors to the terminal:
-    /// always, alwaysansi (always use only ANSI color codes),
+    /// always, always-ansi (always use only ANSI color codes),
```

Comment:
Make sure that the string shown here is an exact match for what the CLI accepts. So if we're making this change, `--color=always-ansi` must work. I believe that previously, `--color=alwaysansi` was required and `always-ansi` was not accepted.